### PR TITLE
(AWSAuthSDK) Upgrade Facebook SDK to v7.1.1

### PR DESCRIPTION
**Pull request by ruiguoamz**
*Issue:*

In file `FBSDKCoreKitImport.h`
the code is like
```swift
#if FBSDK_SWIFT_PACKAGE
#import <FBSDKCoreKit.h>
#else
#import <FBSDKCoreKit/FBSDKCoreKit.h>
#endif
```
but it is build failed because the code runs into `else` block and says `'FBSDKCoreKit/FBSDKCoreKit.h' file not found`. 

The previous code is like:
```swift
#import <FBSDKCoreKit.h>
```
I don't find where does `FBSDK_SWIFT_PACKAGE` defined. I am thinking can I just keep it as the previous way?

*Description of changes:*
- the sample app works (done manual test)

- Update AWSFacebookSignIn.podspec

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: AWSAuthSDK/AWSAuthSDK.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -199,6 +199,21 @@
 		B9AC6319234F00FA00A713A5 /* AWSAuthUIHelper.m in Sources */ = {isa = PBXBuildFile; fileRef = B9AC6317234F00F900A713A5 /* AWSAuthUIHelper.m */; };
 		B9AC631A234F00FA00A713A5 /* AWSAuthUIHelper.h in Headers */ = {isa = PBXBuildFile; fileRef = B9AC6318234F00FA00A713A5 /* AWSAuthUIHelper.h */; settings = {ATTRIBUTES = (Public, ); }; };
 		B9E2F6CD231EEE63009EAA48 /* AWSMobileClientInitializerTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = B9E2F6CC231EEE63009EAA48 /* AWSMobileClientInitializerTests.swift */; };
+		D8FAC273251D1A87005F9FC6 /* FBSDKCoreKitImport.h in Headers */ = {isa = PBXBuildFile; fileRef = D8FAC202251D1A87005F9FC6 /* FBSDKCoreKitImport.h */; };
```
Comment (royjit) [inline AWSAuthSDK/AWSAuthSDK.xcodeproj/project.pbxproj:202]: What are these changes? Below I only see some file modified and no new files.

Comment (ruiguoamz) [reply AWSAuthSDK/AWSAuthSDK.xcodeproj/project.pbxproj:202]: When I finished replacing all header files and opened project file, I found out that not every header files are in the project file. So I dragged them one by one. Some files in v7 are the same with v6, I believe that's the reason you are not seeing new files.

**Comment by palpatim** [comment]
How did you do the manual test? The only way we can validate this from end to end is by publishing the modified SDK to a private spec repo and consuming the updated SDK via CocoaPods. Simply pointing to a local development path isn't sufficient to ensure the podspec is set up correctly.

**Comment by palpatim** [comment]
Related to https://github.com/aws-amplify/aws-sdk-ios/issues/2944

File: AWSAuthSDK/AWSAuthSDK.xcodeproj/project.pbxproj (hunk 1 above)
Comment (palpatim) [reply AWSAuthSDK/AWSAuthSDK.xcodeproj/project.pbxproj:202]: In this case, it looks like the opposite: There were files in the directory tree that were not referenced in the project file. This commit fixes that.

**Review by palpatim — APPROVED** [review]

**Pull request merged**